tests/beacon/helpers.py

```diff
@@ -1,22 +1,36 @@
-from eth.beacon.enums import (
-    ValidatorStatusCode,
+from eth_utils import to_tuple
+
+from eth._utils import bls
+
+from eth.constants import (
+    ZERO_HASH32,
 )
+from eth.beacon.constants import (
+    EMPTY_SIGNATURE,
+)
+from eth.beacon.types.deposit_input import DepositInput
 from eth.beacon.types.validator_records import (
     ValidatorRecord,
 )
 
-from eth_utils import to_tuple
-
 
-def mock_validator_record(pubkey):
+def mock_validator_record(pubkey,
+                          far_future_slot,
```

Comment:
I'm starting to think passing around `far_future_slot` doesn't really make much sense. It is more akin to `GWEI_PER_ETH` and `EMPTY_SIGNATURE` than some of the other configurable constants. Specifically, there is no reason that `far_future_slot` will ever be changed in value unless the bit length of slot is increased or decreased in some deployment.

My opinion is to define as `FAR_FUTURE_SLOT` and remove it from all function params and instead import and use where needed.

Curious to hear both of your thoughts on this @hwwhww and @pipermerriam 

context for @pipermerriam -- `far_future_slot` is a default value used for many of the validator fields and is defined as the max value in `uint64` (`64**2 - 1`). Unlike constants things like `epoch_length` it will never be configured differently depending on the deployment context.

Reply:
@djrtwo Agreed! I think `far_future_slot` could be moved to `eth.beacon.contants`.